.changeset/dry-flies-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+"@jpmorganchase/uitk-lab": minor
+---
+
+Add QA Examples to Progress Component, Add correct font-sizes to Linear Progress
```

Comment:
QA examples isn't that important for changelog (keep if you wish). But should emphasis on the 'fix' not 'add' for font size?